Add docs about the FLARE pattern (#102)
LangStream/documentation: pipeline-agents/ai-actions/compute-ai-embeddings.md

    text: "{{% value }}"
```

### Automatically computing the embeddings over a list of inputs

It is possible to perform the same computation over a list of inputs. This is useful when you want to perform a computation over a list of inputs, for example, a list of questions.
You can take the [Flare pattern](../../building-applications/flare-pattern.md) as an example.

In the example below we use the 'loop-over' capability to compute the embeddings for each document in the list of documents to retrieve.

```yaml
  - name: "compute-embeddings"
    type: "compute-ai-embeddings"
    configuration:
      loop-over: "value.documents_to_retrieve"
      model: "${secrets.open-ai.embeddings-model}"
      embeddings-field: "record.embeddings"
      text: "{{ record.text }}"
```   

> When you use "loop-over" it means that the agent executes for each element in a list instead that operating on the whole message.

Reviewer: ```suggestion
When you use "loop-over", the agent executes for each element in a list instead of operating on the whole message.
```

> You use "record.xxx" in order to refer to the current element in the list.

Reviewer: ```suggestion
Use "record.xxx" to refer to the current element in the list.
```
